### ISSUE TITLE
I cant run my python scripts : ((

### ISSUE THREAD

**Issue by YasTouchedMySpaggett** [bugy/script-server#598]
hi, 
I was trying to run my scripts but i get the following error:
Calling the script...

 ---  ERRORS  --- 
Exec format error

my script is in python and i have all the modules install (libraries) since im able to run the script in the terminal with:
"python3 <scriptname>"
and it runs normally. Sadly i cant post the script code(nda) but im sure it works fine.
My installation is in portainer as a container by it self and the web interface works fine ( even the admin panel works remotely)

Why do i get the error?

**Comment by bugy** [comment]
Hi, please try this one:
https://stackoverflow.com/a/30551364

YasTouchedMySpaggett ***@***.***> schrieb am So., 27. Nov.
2022, 10:23:

**Comment by YasTouchedMySpaggett** [comment]
its not a shell script, just python, and i uploaded the script from the admin panel. So it does not have permission issues.

**Comment by bugy** [comment]
Do you have a shebang in the script?

YasTouchedMySpaggett ***@***.***> schrieb am So., 27. Nov.
2022, 12:05:

**Comment by YasTouchedMySpaggett** [comment]
Sorry for repling so  late and thank you a lot for your time.

No i dont think so, before googling it i didnt even know what it was sadly. Should i add something like that?

**Comment by YasTouchedMySpaggett** [comment]
should i put this? 

#!/usr/bin/env python3

**Comment by bugy** [comment]
Yes, please do it. Otherwise Linux won't know how to execute the script



YasTouchedMySpaggett ***@***.***> schrieb am So., 27. Nov.
2022, 18:32:

**Comment by MNeill73** [comment]
The shebang statement specifies which interpreter should be run to parse 
the script.

Personally I've never seen the "env interpreter" syntax before very 
recently. Historically, it's /path/to/interpreter, so, /bin/bash (because I 
always invoke the statically-linked version of shells), or 
/usr/bin/python27 (specific python version), or /usr/bin/python (to invoke 
the "current" python interpreter by way of soft link).

**Comment by YasTouchedMySpaggett** [comment]
thanks a lot, seems to be working now, how do i install modules on the specific pyhton3? even tho my python has all the modules the "env" used by #!/usr/bin/env python3 does not have all my modules

**Comment by YasTouchedMySpaggett** [comment]
also i have it in a container, maybe i need to install the packages there? i use portainer

**Comment by bugy** [comment]
Yes, you need to install packages there

**Comment by MNeill73** [comment]
In the container, and invoke the appropriate and specific pip to install 
them in the modules directory for that version (I.e. if /usr/bin/python is 
linked to python37, then specifically invoke pip37 instead of plain pop to 
make sure you have them in the python37 includes).
